src/tools/ping/Server.cpp

```diff
@@ -78,79 +78,35 @@ struct PingServer {
 
 void QuicPingServerRun()
 {
+    QUIC_SETTINGS Settings{0};
```

Comment:
Why not use the `.hpp` interface?

Reply:
No. I don't want to add a new dependency here.